Source/WebCore/css/CSSProperties.json

```diff
@@ -116,11 +116,13 @@
         "Indicates that this CSS property is descriptor only (populates.",
         "CSSProperty::isDescriptorOnly().",
         "",
-        "",
         "* font-property:",
         "Indicates that this CSS property is font-related. It must have corresponding",
         "methods on the FontDescription class.",
         "",
+        "* font-subproperty:",
+        "Indicates that this CSS property is a subproperty of the font shorthand.",
+        "",
```

Comment:
I have a slight preference to not add new fields (given there's quite a bit of them) and to use the list of longhands from `font`. It's also unclear how/why it differs from `font-property`. I've been wanting to clean some of these (e.g. high/top/sink-priority for instance).

Reply:
I think it’s tricky to use the list of longhands but I will do it if I can figure it out. If not I will return and switch to that.

Reply:
OK, after a little bit of a journey learning Python, this is done.